examples/game/breakout.rs

```diff
@@ -5,268 +5,381 @@ use bevy::{
     sprite::collide_aabb::{collide, Collision},
 };
 
-/// An implementation of the classic game "Breakout"
-const TIME_STEP: f32 = 1.0 / 60.0;
+use components::*;
+use resources::*;
+
+/// Constants that can be used to fine-tune the behavior of our game
+mod config {
+    use bevy::render::color::Color;
+    use bevy::ui::Val;
+    // TODO: add various Vec2's and Transforms to this config module for clarity and consistency
+    // Blocked on https://github.com/bitshifter/glam-rs/issues/76
```

Comment:
you can use `const_vec2!` https://docs.rs/glam/0.14.0/glam/macro.const_vec2.html to create `const Vec2`. There are const macros for most types

Reply:
Done :) const `Transform` macros were too hard for me to tackle, especially in this PR, so I made #2097 to track it and left a note in the code.